### ISSUE TITLE
Telemetry- statsd Consul doesn't send data upon container restarts !

### ISSUE THREAD

**Issue by srkurapa** [hashicorp/consul#3360]
Is there any restart order or priority while using telegraf , Influxdb and grafana for monitoring consul ( Version 8 ) ? 
Looks like the consul metrics are not being sent automatically to telegraf upon container restarts. 
I would start seeing the cluster info only after a manual consul cluster restart. 
( Using dogstatsd with port 8125 in consul configuration to send metrics to telegraf )

**Comment by slackpad** [comment]
Hi @srkurapa this looks like an existing library issue. Closing as a duplicate of https://github.com/armon/go-metrics/issues/39.
